rpcs3/Emu/RSX/GCM.cpp

```diff
@@ -773,6 +798,40 @@ std::string rsx::get_method_name(const u32 id)
 	return fmt::format("unknown/illegal method [0x%08x]", id);
 }
 
+Surface_antialiasing to_surface_antialiasing(u8 in)
+{
+	switch (in)
+	{
+	case 0: return Surface_antialiasing::center_1_sample;
+	case 3: return Surface_antialiasing::diagonal_centered_2_samples;
+	case 4: return Surface_antialiasing::square_centered_4_samples;
+	case 5: return Surface_antialiasing::square_rotated_4_samples;
+	}
+	throw EXCEPTION("unknow surface antialiasing format %x", in);
+}
+
+Surface_color_format to_surface_color_format(u8 in)
```

Comment:
twell, if it makes sense and easier to use some other random enums instead of cell gcm enums, that's alright. But why magic numbers in convert functions? It's hard to see if this case 0 is really cell gcm what it used to be or not.